src/components/Calculator.jsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react';
+import calculate from './logic/calculate';
+
+function Calculator() {
+  const [btn, setBtn] = useState('');
+  const [obj, setObj] = useState({});
+  const [calc, setCalc] = useState({});
+
+  const handleButtonClick = (e) => {
+    let valeur = '';
+    setBtn((p) => {
+      valeur = p + e.target.value;
+      setObj({ ...obj, next: parseInt(valeur, 10) });
+      return valeur;
+    });
+  };
+
+  const handleSigns = (e) => {
+    setObj({ ...obj, total: parseInt(btn, 10), operation: e.target.value });
+    setBtn('');
+  };
+
+  const handleEqual = (e) => {
+    setCalc(() => ({ ...calc, obj, buttonName: e.target.value }));
+    if (calc.obj !== undefined) {
+      const num = calculate(calc.obj, calc.buttonName);
+      setBtn(num.total);
+      setCalc({});
+    }
+  };
+
+  const handleInput = (e) => {
+    setBtn(e.target.value);
+  };
+
+  return (
+    <div className="calc">
+      <input value={btn} onChange={handleInput} className="inpute" />
+      <input type="button" onClick={handleSigns} className="signs gray" value="A/C" />
+      <input type="button" onClick={handleSigns} className="signs gray" value="+/-" />
+      <input type="button" onClick={handleSigns} className="signs gray" value="%" />
+      <input type="button" onClick={handleSigns} className="signs orange" value="÷" />
+      <input type="button" onClick={handleButtonClick} className="signs gray" value="7" />
+      <input type="button" onClick={handleButtonClick} className="signs gray" value="8" />
+      <input type="button" onClick={handleButtonClick} className="signs gray" value="9" />
+      <input type="button" onClick={handleSigns} className="signs orange" value="*" />
```

Comment:
- [ ] When performing calculations that involve dividing by 0 or modulo by 0 I should get the error messages in the operate.js file: "Can't divide by 0." and "Can't find modulo as can't divide by 0." But I get nothing. Kindly fix that.